src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import styles from './Toast.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { CloseIconButton } from './CloseIconButton/CloseIconButton';
+
+export enum TOAST_SKIN {
+  status = 'status',
+  success = 'success',
+  error = 'error',
+}
+
+export interface ToastProps {
+  /** Define the styles for toast */
+  skin: TOAST_SKIN;
+  shouldShowCloseButton?: boolean;
+  /** Callback function, will be called when click on close button */
+  onClose?(): void;
+}
+
+export interface DefaultProps {
+  shouldShowCloseButton: boolean;
+}
+
+interface State {}
+
+export class Toast extends React.Component<ToastProps, State> {
+  static displayName = 'Toast';
+
+  static defaultProps: DefaultProps = {
+    shouldShowCloseButton: false,
+  };
+
+  handleOnCloseClick = () => {
+    const { onClose } = this.props;
+    onClose && onClose();
+  };
+
+  render() {
+    const { skin, shouldShowCloseButton, children, ...rest } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile, rtl }) => (
+          <div
```

Comment:
I think the root element should have `role="alert"` so it will create an `aria-live` region
(it indicates to screen readers that the text here important and they will watch for changes in this element)
You can read more about it in the [spec](http://w3c.github.io/aria-practices/#alert)

Reply:
The `rtl` state can be removed from here